Cargo.toml

```diff
@@ -18,6 +18,7 @@ members = [
     'examples/opengl_underlay',
     'examples/opengl_texture',
     'examples/ffmpeg',
+    'examples/gstreamer_slint_demo',
```

Comment:
I suggest to just call it `gstreamer`

Reply:
Tools were having issues due to the name collision with the gstreamer crate. I was working in vscode. I think I just had the example loaded in the IDE. Helps humans too IMO. That probably affects the ffmpeg example code too, but may depend on IDE and IDE config. Still want it "gstreamer"?

Reply:
Fair point. How about `gstreamer-player`?

Reply:
I'm OK with the name change, but if we call it a player, I should at least put a warning about the buttons not working in the example README.

Reply:
Sounds good to me.